src/TestFramework/TestFrameworkExtraOptionsFilter.php

```diff
@@ -43,37 +43,21 @@
 /**
  * @internal
  */
-abstract class TestFrameworkExtraOptions
+final class TestFrameworkExtraOptionsFilter
 {
     /**
-     * @var string
+     * @param string[] $initialRunOnlyOptions
+     *
+     * @throws \Safe\Exceptions\PcreException
```

Comment:
should use import statements here, but I would argue those exceptions should not be shown